### DISCUSSION TITLE
layer=other ?

### DISCUSSION THREAD

**Discussion by yohanboniface** [komoot/photon#820] [Q&A]
Hi there :)

The README seems to indicate a `layer=other` value:

![image](https://github.com/komoot/photon/assets/146023/2659c4e0-8c4b-4a1c-a4b0-375e73308edb)

But this call to the API says nope:

https://photon.komoot.io/api?q=parc%20&layer=other

```json
{"message":"Invalid layer 'other'. Allowed layers are: house,street,locality,district,city,county,state,country"}
```

Who to believe ? :)

My use case was to allow to search for natural parcs (in addition to `city and `county`).

Thanks!

Yohan

**Comment by lonvia** [comment]
You are looking at the README of the master version. Please refer to the [README of the latest release](https://github.com/komoot/photon/tree/0.5.0) for the documentation of the API on photon.komoot.io.

Or to put it differently: the 'other' layer is a new not yet release feature.

**Reply by yohanboniface** [reply]
Haaa, my bad, sorry for the noise! :)

**Reply by yohanboniface** [reply]
(A good clue though that this yet to be released feature will be used! :) )

**Reply by yohanboniface** [reply]
Hey Sarah :)

Any news about `layer=other` being deployed in production servers ? :)

Also, I wonder what else will be targeted by `layer=other` ? In my case, users feedback seems to point to a need for `layer=natural`, dunno if that makes sense for you :)

**Reply by lonvia** [reply]
Thanks for the reminder. I thought I had updated the machine. It's now running the newest 0.6.1.

> Also, I wonder what else will be targeted by layer=other ?

It's mostly natural features these days: waterways, mountain peaks, bays etc.

In the future some larger man made features like railways might make it in there as well. And maybe features like hiking and cycling routes, if they ever make it into Photon.

**Reply by yohanboniface** [reply]
> It's now running the newest 0.6.1.

Yay, thank you!

To give a bit more context, in uMap, we use Photon twice: once as the generic search engine, and once for the overpass import helper:

![image](https://github.com/user-attachments/assets/bd29aced-17db-4331-ac73-9e227b858dd1)

As you can see, there is a search input, that will be used to limit the overpass query, so in this case, what we'd really need is whatever OSM feature which is an area (vs a node or a line). Is that something possible already or would that make sense ?

**Reply by lonvia** [reply]
Photon doesn't have geometry information about the OSM object, so couldn't do that for you.

**Reply by yohanboniface** [reply]
Yes, I know that :)
I don't need the geometry, I just need the id, of a feature that is an area on OSM. :)
